src/ripple/rpc/handlers/LedgerEntry.cpp

```diff
@@ -344,6 +345,168 @@ doLedgerEntry(RPC::JsonContext& context)
             jvResult[jss::error] = "malformedRequest";
         }
     }
+    else if (context.params.isMember(jss::bridge_account))
+    {
+        expectedType = ltBRIDGE;
+        auto const& jsBridgeAccount = context.params[jss::bridge_account];
+        if (!jsBridgeAccount.isString())
+        {
+            uNodeIndex = beast::zero;
+            jvResult[jss::error] = "malformedRequest";
+        }
+        else if (auto const account =
+                     parseBase58<AccountID>(jsBridgeAccount.asString());
+                 account && account->isNonZero())
+        {
+            Keylet keylet = keylet::bridge(*account);
+            uNodeIndex = keylet.key;
+        }
+        else
+        {
+            uNodeIndex = beast::zero;
+            jvResult[jss::error] = "malformedRequest";
+        }
+    }
+    else if (context.params.isMember(jss::xchain_claim_id))
```

Comment:
Do these names need to be updated to the new naming schemes? e.g. `jss::xchain_owned_claim_id`

Reply:
Fixed in a536f114d6 [fold] Rename xchain_claim_id and xchain_create_account_claim_id to add "owned" to the name https://github.com/XRPLF/rippled/pull/4292#pullrequestreview-1298363200